.github/workflows/maven-checks.yml

```diff
@@ -41,6 +41,6 @@ jobs:
       - name: Maven Checks
         run: |
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
-          ./mvnw install -B -V -T C1 -DskipTests -Dmaven.javadoc.skip=true -P ci -pl '!presto-test-coverage,!:presto-server'
```

Comment:
Why add this change?

Reply:
With the PDF generation bundled for `presto-docs`, there requires a step to [install necessary tools](https://github.com/prestodb/presto/pull/21883/commits/bdd772432a212c916ac7a642f8673a95b18c90f8) but the step is only added in `docs.yml`, which is already a workflow dedicated to building docs. 

If I added the LaTeX tooling installation in `maven-checks.yml`, it will be used for `presto-docs` only.